global.json

```diff
@@ -1,4 +1,9 @@
 {
+  "sdk": {
+    "version": "3.0.100",
```

Comment:
This will require devs to have that version of a later version installed locally. In example when opening with VS or directly `dotnet msbuilding` a project. That shouldn't be a problem though.